test/fixtures/key/expected.js

```diff
@@ -1,4 +1,10 @@
 "use strict";
 
-elementOpen("div", "1");
+elementOpen("div");
+elementOpen("div", "1", ["key", "1"])
+elementClose("div")
```

Comment:
Excellent, that's exactly what I needed to know.

Now, any way to preserve the indentation? It's minor, but makes reading the output much easier.